pyfar/plot/_line.py

```diff
@@ -77,6 +90,21 @@ def _freq(signal, dB=True, log_prefix=None, log_reference=1, freq_scale='log',
     else:
         data = np.abs(signal.freq)
 
+    if not type(signal) is FrequencyData and signal.complex:
+        if side == 'right':
+            data = dsp.fft.remove_mirror_spectrum(data, force=True)
+            frequencies = np.atleast_1d(
+                dsp.fft.rfftfreq(signal.n_samples, signal.sampling_rate))
+        elif side == 'left':
+            dc_idx = data.shape[-1] // 2
+            data = data[..., dc_idx-1:]
+            frequencies = np.atleast_1d(
+                dsp.fft.rfftfreq(signal.n_samples, signal.sampling_rate))
+        else:
+            ValueError('Invalid side parameter, pass left or right ...')
+    else:
+        frequencies = signal.frequencies
+
```

Comment:
We could discuss to make this more explicit

- use 'auto' as the default value for `side`
- mention in the docs, that side can be 'left' or 'right' in case of complex signals (default 'right')
- throw an error if side is not 'auto' in case of `FrequencyData` and real valued `Signal` objects.

Probably, we could have a private function for this to avoid rewriting code. It could get the data (magnitude, phase, group delay) and type(signal).

Reply:
I shifted everything into a private method in _utils. Do you think we really need the mask as discussed in today's meeting. I think everything is implicitly covered by the is_complex check, isn't it?